docs/guides/sdks/android-sdk-reference/android-sdk-reference-appsflyerlib.md

```diff
@@ -496,6 +492,74 @@ This function takes no parameters.
 
 **Usage example**
 
+### logAdRevenue
+(Supported from SDK v.6.15.0)
+
+**Method signature**
+
+```objectivec
+-(void)logAdRevenue:(AFAdRevenueData *)adRevenueData additionalParameters:(NSDictionary * **_Nullable**)additionalParameters;
+```
+**Description**
+
+The method sends an ad revenue event to AppsFlyer. See more information in [Ad revenue](doc:ad-revenue-2).
+
+
+**Input parameters**
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `adRevenueData`* | [`AFAdRevenueData*`](#afadrevenuedata) | An object that encapsulates all the mandatory parameters of the adRevenue event. The object is passed to the logAdRevenue method. |
+| `additionalParameters` | NSDictionary * _Nullable | An optional dictionary containing additional parameters to log with the adRevenue event. |
+
+**Returns**
+
+`void`.
+
+#### AFAdRevenueData
```

Comment:
In [validateAndLogPurchase](https://dev.appsflyer.com/hc/docs/ios-sdk-reference-appsflyerlib#afsdkpurchasedetails), we decided to include it on the same topic because it is an auxiliary object not used by any other function. 
WDYT?

Reply:
@dori-af makes sense. Agreed